{
    "pattern": "USE test; CREATE TABLE t1 (f1 INT <constraint_type_parameter>, f2 INT <constraint_type_parameter>, <constraint_enforce_replace>, <constraint_enforce_replace>) ENGINE=<engine_parameter>; INSERT INTO t1 (f1) VALUES <dml_values_parameter>; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; INSERT INTO t1 (f1) SELECT f1 FROM t1; UPDATE t1 SET f2=<numeric_function_parameter>(); ANALYZE TABLE t1; SHOW INDEXES FROM t1; SELECT COUNT(DISTINCT f1), COUNT(DISTINCT f2), COUNT(DISTINCT f1, f2), COUNT(*) FROM t1; SELECT * FROM mysql.innodb_index_stats;",
    "applied_rules": [
        "<constraint_type_parameter>",
        "<constraint_enforce_replace>",
        "<dml_values_parameter>",
        "<numeric_function_parameter>"
    ],
    "Affected Elements": "SHOW INDEXES, ANALYZE TABLE",
    "Root Cause Analysis": "The cardinality reported by SHOW INDEXES does not match the correct statistics in mysql.innodb_index_stats due to a bug in the InnoDB storage engine."
}